lib/main_containers.pm

```diff
@@ -100,22 +100,33 @@ sub load_host_tests_load_containerd_nerdctl {
 
 sub load_container_tests {
     my $runtime = get_required_var('CONTAINER_RUNTIME');
+    $runtime =~ tr/ //ds;
+    my @runtimes = split(',', uc $runtime);
+
     if (get_var('BOOT_HDD_IMAGE')) {
         loadtest 'installation/bootloader_zkvm' if is_s390x;
         loadtest 'boot/boot_to_desktop' unless is_jeos;
     }
 
     if (is_container_image_test()) {
-        # Container Image tests
+        # Container Image tests common
         loadtest 'containers/host_configuration' unless (is_expanded_support_host || is_ubuntu_host || is_jeos);
-        load_image_tests_podman() if ($runtime =~ 'podman');
-        load_image_tests_docker() if ($runtime =~ 'docker');
-    } else {
-        # Container Host tests
-        load_host_tests_podman() if ($runtime =~ 'podman');
-        load_host_tests_docker() if ($runtime =~ 'docker');
-        load_host_tests_load_containerd_crictl() if ($runtime =~ 'containerd_crictl');
-        load_host_tests_load_containerd_nerdctl() if ($runtime =~ 'containerd_nerdctl');
     }
+
+    foreach (@runtimes) {
+        if (is_container_image_test()) {
+            # Container Image tests
+            load_image_tests_podman() if ($_ =~ 'podman');
+            load_image_tests_docker() if ($_ =~ 'docker');
```

Comment:
I would prefer to go either string comparison `eq` or match regex literals (maybe even case-insensitive)

If we go the second way, we can also omit `$_`.

```
... if (/podman/i);
``` 

Should work

Reply:
This is implemented too